functions/go/fmt/README.md

```diff
@@ -0,0 +1,73 @@
+# fmt
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources using openapi schema.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Format formats the field ordering in YAML configuration files. Field ordering
+roughly follows the ordering defined in the source Kubernetes resource definitions
+(i.e. go structures), falling back on lexicographical sorting for unrecognized fields.
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+#### Format a package
+
+Let's start with the input resource in a package
+
+```yaml
```

Comment:
Why not moving this into `examples` and have it always tested.

Reply:
I will add an example in a separate PR. This shows up as help output.

Reply:
We are starting to enforcing some rules in CI checks in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/246
One of them is that each function should have at least one example.
I'd suggest you to add a simple example in this PR.

Reply:
Nice. Updated.